docs/src/pages/components/cards/PlayerCard.js

```diff
@@ -109,15 +110,18 @@ export default function PlayerCard() {
   return (
     <ThemeProvider theme={theme}>
       <Card variant="outlined" sx={{ display: 'flex', p: 1 }}>
-        <img
-          alt="Beside Myself album cover"
-          style={{ borderRadius: 8 }}
+        <Avatar
+          variant="rounded"
+          alt="Song cover"
           src="/static/images/cards/basement-beside-myself.jpg"
-          width="100"
-          height="100"
+          imgProps={{
+            width: '124',
+            height: '124',
+          }}
+          sx={{ width: 124, height: 124 }}
```

Comment:
This reverts https://github.com/mui-org/material-ui/pull/27675.

I don't think this PR should touch files outside of the rebranding.

Reply:
there is some minor design change after the previous PR. will use the `alt` from #27675.

Reply:
> PlayerCard is for rebranding.

It is not. This is a demo. Inline the things you need. Demos are not meant to be embedded in any other context than demos.

Reply:
> Demos are not meant to be embedded in any other context than demos

Thanks for the clarification, will move those intended rebranding components outside of demos.